float64[] sim_state
string node_name
---
